test/cookbooks/mysql_tuning_test/recipes/ohai_plugin.rb

```diff
@@ -19,5 +19,11 @@
 # limitations under the License.
 #
 
+# Silence extraneous stderr from ohai
+# "INFO: The plugin path /etc/chef/ohai/plugins does not exist. Skipping..."
+directory '/etc/chef/ohai/plugins' do
+  recursive true
+end
```

Comment:
Maybe we should move this to the `mysql_tuning::ohai_plugin` recipe, shouldn't we?